app/assets/javascripts/charts/growth.js

```diff
@@ -4,36 +4,30 @@ var GrowthChart = function (element, data) {
     title: {
       text: 'Growth Histogram For Measurement ' + data.processed_file_id
     },
-    xAxis: [{
```

Comment:
I did NOT review this file as it's isolated and as the domain expert I trust you know what you want here :)